src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/IntegrationTestBackend.scala

```diff
@@ -29,7 +29,7 @@ private[spark] trait IntegrationTestBackend {
 }
 
 private[spark] object IntegrationTestBackendFactory {
-  def getTestBackend(): IntegrationTestBackend = {
+  def getTestBackend: IntegrationTestBackend = {
     Option(System.getProperty("spark.kubernetes.test.master"))
```

Comment:
Should we instead check deployMode here? There's a separate flag for this now.

Reply:
Yes, done.